modular_zubbers/modules/modular_weapons/code/company_and_or_faction_based/carwo_defense_systems/ammo/carwo.dm

```diff
@@ -0,0 +1,20 @@
+/obj/projectile/bullet/c35sol
+	damage = 20
+	wound_bonus = -10
+	wound_falloff_tile = -10
+
+/obj/projectile/bullet/c40sol
+	damage = 30
```

Comment:
I have to wonder why an ammo type for a gun you can carry several magazines for does more raw damage by default than any bulky laser gun?

Reply:
Apparently it's balanced so far on DPS, which actually puts it lower than the WTs.

Reply:
The C40 Sol rifle has slightly less DPS than the WT-551, it's not a major difference but y'know